src/app/features/owner-meals/owner-meals.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { AddingModalComponent } from 'src/app/shared/adding-modal/adding-modal.component';
+import { Meal } from 'src/app/core/interfaces/meal';
+import { Subscription } from 'rxjs';
+import { OwnerMealsService } from '../../core/services/owner-meals.service';
+import { ActivatedRoute } from '@angular/router';
+@Component({
+  selector: 'owner-meals',
+  templateUrl: './owner-meals.component.html',
+  styleUrls: ['./owner-meals.component.scss'],
+})
+export class OwnerMealsComponent implements OnInit {
+  public ownerMealsSubscription: Subscription;
+  public meals: Meal[] = [];
```

Comment:
Why are meals and ownerMealsSubscription public, and why does restaurantId miss a 'public' or a 'private', please research that